pipelines/introduction-to-generic-pipelines/README.md

```diff
@@ -42,7 +42,9 @@ In this introductory tutorial you will learn how to create a generic pipeline an
 ### Setup
 
 This tutorial uses the `introduction to generic pipelines` sample from the https://github.com/elyra-ai/examples GitHub repository.
-1. Launch JupyterLab.
+1. Launch JupyterLab.    
+   > **Note**: When you start JupyterLab using the `jupyter lab` command, it loads the contents of the current working directory.  
+   > For examples, we recommend creating a new directory on your system and running the `jupyter lab` command from that directory.  
```

Comment:
I believe the motivation behind this extra info is to also to make sure the root directory used to run juypterlab is not an existing git repository, otherwise the next step below won't work. Maybe it's worth mentioning that somehow here.

Reply:
Yes!